PrebidMobile/API1.0/src/main/java/org/prebid/mobile/NativeAdUnit.java

```diff
@@ -0,0 +1,197 @@
+package org.prebid.mobile;
+
+import android.support.annotation.NonNull;
+
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+/**
+ * For details of the configuration of native imps, please check this documentation:
+ * https://www.iab.com/wp-content/uploads/2018/03/OpenRTB-Native-Ads-Specification-Final-1.2.pdf
+ */
+public class NativeAdUnit extends AdUnit {
+    enum NATIVE_REQUEST_ASSET {
+        TITLE,
+        IMAGE,
+        DATA
+    }
+
+    // constants
+
+    static String VERSION = "ver";
+    static String SUPPORTED_VERSION = "1.2";
+    static String CONTEXT = "context";
+    static String CONTEXT_SUB_TYPE = "contextsubtype";
+    static String PLACEMENT_TYPE = "plcmttype";
+    static String PLACEMENT_COUNT = "plcmtcnt";
+    static String SEQ = "seq";
+    static String ASSETS = "assets";
+    static String A_URL_SUPPORT = "aurlsupport";
+    static String D_URL_SUPPORT = "durlsupport";
+    static String EVENT_TRACKERS = "eventtrackers";
+    static String PRIVACY = "privacy";
+    static String EXT = "ext";
+    static String EVENT = "event";
+    static String METHODS = "methods";
+    static String LENGTH = "len";
+    static String REQUIRED = "required";
+    static String ASSETS_EXT = "assetExt";
+    static String WIDTH_MIN = "wmin";
+    static String HEIGHT_MIN = "hmin";
+    static String WIDTH = "W";
+    static String HEIGHT = "h";
+    static String TYPE = "type";
+    static String MIMES = "mimes";
+    static String TITLE = "title";
+    static String IMAGE = "img";
+    static String DATA = "data";
+    static String NATIVE = "native";
+    static String REQUEST = "request";
+
+
+    public NativeAdUnit(@NonNull String configId) {
+        super(configId, AdType.NATIVE);
+    }
+
+    HashMap<String, Object> requsetConfig = new HashMap<>();
+
+    public void setContext(Integer contextId) {
```

Comment:
I propose to use `setContextTypeId` to not mix `Android Context` and `NativeAdUnit Context`